projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -167,6 +175,39 @@ const validateData = ( data: SeriesData[] ) => {
 	return null;
 };
 
+const HighlightTooltip: React.FC< {
+	series: SeriesData[];
+	selectedIndex: number | undefined;
+} > = ( { series, selectedIndex } ) => {
+	const tooltipContext = useContext( TooltipContext );
```

Comment:
The component `HighlightTooltip` has to be introduced because the `TooltipContext` is available inside the `XYChart`, I am hoping to move this into a wrapper component for Tooltip to reuse the logic for other chart types